app/packs/controllers/auto_citation_controller.js

```diff
@@ -72,17 +72,17 @@ export default class extends Controller {
     return this.titleFieldTarget.value
   }
 
-  get year() {
-    if (this.hasEmbargoTarget && this.embargoTarget.checked && this.yearTarget.value.length < 1) {
+  get embargoYear() {
+    if (this.hasEmbargoTarget && this.embargoTarget.checked) {
       return this.embargoYearTarget.value
     }
-    return this.yearTarget.value
+    return null
   }
 
   get date() {
     const date = new Date();
-    if (this.year) {
-      date.setYear(this.year)
+    if (this.embargoYear) {
+      date.setYear(this.embargoYear)
```

Comment:
Should it be set to current year if the embargo date is not present?

Reply:
It is when `date` is initialized.